base.yaml

```diff
@@ -123,22 +96,21 @@ catalog_server:
 
 postgresql:
   _install: true
+  replication:
+    enabled: false
```

Comment:
Why disable replication? It's a simple and easy way to have backup of the data and also balance the read load workloads.

Reply:
The number of postgres databases including replicas is exhausting the CPU quota of a single node. I had to increase from 4 to 8 cores to just get a basic set of services. For a single-host deployment, I also think that replication is really not necessary, a first resolution would be to actually use backups. In any case, it's included in `base.yaml` because it would be one of the first parameters to edit if a larger cluster is available.